src/app/config/config.c

```diff
@@ -8177,10 +8177,9 @@ getinfo_helper_config(control_connection_t *conn,
       switch (var->type) {
         case CONFIG_TYPE_STRING: type = "String"; break;
         case CONFIG_TYPE_FILENAME: type = "Filename"; break;
-        case CONFIG_TYPE_UINT: type = "Integer"; break;
+        case CONFIG_TYPE_POSINT: type = "Integer"; break;
         case CONFIG_TYPE_UINT64: type = "Integer"; break;
         case CONFIG_TYPE_INT: type = "SignedInteger"; break;
-        case CONFIG_TYPE_PORT: type = "Port"; break;
```

Comment:
These changes delete `CONFIG_TYPE_PORT`. Are we still using it elsewhere? (We don't seem to be, but I could have missed something).

Reply:
We aren't using it anywhere else, no.